app/assets/javascripts/src/Metamaps.Map.js

```diff
@@ -49,12 +49,18 @@ Metamaps.Map = {
       return false
     })
 
+    $('.starMap').click(function () {
+      if ($(this).text() === 'star') self.star()
+      else self.unstar()
```

Comment:
:'( I might look into if this can be done more nicely with react. I'll need to figure out if the state needs to be in sync with Backbone, or just with the server. It's more annoying if it needs to sync with backbone